cmd/algod/main.go

```diff
@@ -232,9 +238,28 @@ func run() int {
 					telemetryConfig.SessionGUID = *sessionGUID
 				}
 			}
-			err = log.EnableTelemetry(telemetryConfig)
+			// Try to enable remote telemetry now when URI is defined. Skip for DNS based telemetry.
+			ctx, cancel := context.WithTimeout(context.Background(), DefaultStaticTelemetryStartupTimeout)
+			defer cancel()
```

Comment:
```suggestion
			ctx, _ := context.WithTimeout(context.Background(), DefaultStaticTelemetryStartupTimeout)
```
no need to cancel I guess?